.gitignore

```diff
@@ -238,3 +238,7 @@ $RECYCLE.BIN/
 *.lnk
 
 # End of https://www.toptal.com/developers/gitignore/api/macos,windows,linux,node,visualstudiocode
+
+# Editor-specific folders
+.obsidian/
+.vscode/
```

Comment:
Put these on top of the `.gitignore` file so that they come above the generated section, making it easier to read the file.